docs/pages/admin-guides/teleport-policy/database-access-controls.mdx

```diff
@@ -0,0 +1,120 @@
+---
+title: Teleport Policy and Database Access Controls
+description: Describes how to import and visualize advanced Teleport Database Access Controls policies using Teleport Policy and Access Graph.
+---
+
+Advanced Database Access Controls features allow Teleport administrators to configure user permissions for individual database objects using flexible, label-based approach for expressing access policies.
+
+The database objects-level access information is automatically synchronized to Teleport Policy, making it possible to see who has particular levels of access to the different parts of the database.
```

Comment:
```suggestion
Database object-level access information is automatically synchronized to Teleport Policy, making it possible to see who has particular levels of access to the different parts of a Teleport-protected database.
```

Since we haven't specified a database yet.